commit 1be1d019a0759bb2554123cd6d8553b99eb1d109 (refs/changes/87/3887/3)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-05-02T09:08:21-07:00 (1 year, 5 months ago)
    
    TRITON-356 VMAPI should use node v6.x
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/Makefile b/Makefile
index e8bbe11..e617edb 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -24,7 +24,7 @@
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v6.14.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than sdc-minimal-multiarch-lts@15.4.1
diff --git a/package.json b/package.json
index fad45ac..9042f19 100644
--- a/package.json
+++ b/package.json
@@ -1,14 +1,14 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.5.1",
+  "version": "9.5.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
     "assert-plus": "1.0.0",
     "async": "0.7.0",
     "backoff": "2.5.0",
-    "bunyan": "1.8.1",
+    "bunyan": "1.8.12",
     "changefeed": "1.3.0",
     "cueball": "2.2.7",
     "dashdash": "1.14.1",
